### ISSUE TITLE
Can Image sizes be changed when viewing (Maps / IMages)

### ISSUE THREAD

**Issue by mikehjones** [bibletime/bibletime#351]
When viewing Images or Maps from sources in repositories (Maps / Other images) in BT, the image is displayed full size. There does not seem to be scroll controls (horizontal) or any way to resize. As is, images are not very useful as the entire images cannot be viewed correctly. I am using Ubuntu 20.04 LTS with KDE.

Is there currently a way to perform a resize on the said images, or perhaps this can be converted to a feature request. Thank you.

**Comment by mikehjones** [comment]
As a side note, the images and maps will display properly when that screen is the only screen open. If other panes are open then no go. This is also an issue with the jpeg files for the timelines. Nothing will make them display (that I have found) completely in an open pane, and also cannot be scrolled. Small in the big scheme of things, but annoying.

**Comment by jaakristioja** [comment]
I can confirm that the images are not properly scrollable, and I think this can be considered to be a bug in the UI due to the severity of this hindering the proper viewing of the images. Thanks for reporting this!

> As a side note, the images and maps will display properly when that screen is the only screen open. If other panes are open then no go. This is also an issue with the jpeg files for the timelines. Nothing will make them display (that I have found) completely in an open pane, and also cannot be scrolled. Small in the big scheme of things, but annoying.

I'm not sure I understand correctly. Did you mean that the images will only be partially displayed when multiple works are open in the same BibleTime subwindow? Also please pardon my ignorance, but what are "jpeg files for the timelines"? Is this in relation with a specific works you have installed in BibleTime? If possible, please state the works/modules you are using, or post screenshots to describe those issues. Thank You! :-)

**Comment by mikehjones** [comment]
Yes. When multiple sub windows, images (such as maps, Dore woodcuts), are viewed in the sub window, but not in entirety, and no horizontal scrolling is provided. The sub window can be enlarged by dragging and eventually the entire image will be viewable, but the sub window will overlay other open windows.

When using the "Lineage" module, which consists of a collection of JPEG images of various lines of lineages, the images are displayed so large, and only vertical scrolling is used, even with the dragging the sub window, there is still no room to view the entire image. Now this may be more of an issue with the original JPEG file, but still if horizontal scrolling was enabled, this issue would be solved.

Thank you.
